### PR TITLE
Squad transfer tablet

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request
SLs get a squad transfer tablet they can use to ask people to swap squad

## Why It's Good For The Game

Reasins

## Changelog
:cl:
add: Added squad transfer tablet for SLs
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
Needs custom squads to compile btw

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -193,6 +193,7 @@
 					vended_items += new /obj/item/radio/headset/mainship/marine(loc, H.assigned_squad, vendor_role)
 					if(istype(H.job, /datum/job/terragov/squad/leader))
 						vended_items += new /obj/item/hud_tablet(loc, vendor_role, H.assigned_squad)
+						vended_items += new /obj/item/squad_transfer_tablet(loc)
```
Comment (Lumipharon) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:196]: player issue, but doesn't this mean any sl can jsut fuck with every squad?

**Review by Lumipharon — APPROVED** [review]

**Comment by Lumipharon** [comment]
I think this needs some restrictions so 1 person can't just mess with all the squads remotely but eh

**Comment by psykzz** [comment]
Checked out the PR - Few pieces of feedback.

 - There is no feedback to the SL about anyone rejecting the request.
     - Additional to this, there is no rate limiting or a way for someone to just always deny requests.
 - The messages generally is missing any flavour.
 - The typical popup (between default squads) looks pretty bad.
image
Can we improve the message a little providing the description optionally if there is one.

**Comment by psykzz** [comment]
Additionally to Lumi's point, i don't mind this allowing one person to submit requests for everyone, but i do think it shouldn't be possible for anyone to use the tablet, even if it won't show anything for some of them.

But we should make sure squad members can't use the tablet.

**Pull request merged**